docs/data-management/mriqc.md

```diff
@@ -10,10 +10,96 @@
-- [ ] Schedule an extra session after the initially-planned scanning period to reacquire it if either the dMRI or the RSfMRI quality is insufficient.
+
+## Visualizing *MRIQC*'s individual reports
+
+Following our protocols<sup>[1]</sup>, the quality of unprocessed images MUST be assessed before executing any processing (e.g., *fMRIPrep* or *dMRIPrep* for preprocessing).
+In addition, *MRIQC* is executed prior any further processing step considering our preliminary findings regarding *defacing*<sup>[2]</sup>.
+
+### Assessing anatomical images (T<sub>1</sub>-weighted and T<sub>2</sub>-weighted)
+- [ ] Open each *MRIQC* report on a current Web Browser (*Google Chrome* is preferred).
+- [ ] Visualize the first mosaic (background-enhanced mosaic) and apply the [exclusion criteria](qaqc-criteria.md#background-enhanced-mosaic)
+- [ ] Scroll down to the zoom in the brain mask mosaic and apply the [exclusion criteria](qaqc-criteria.md#zoomed-in-brain-mosaic)
+- [ ] Assign a quality rating and indicate artifacts with the *Rating widget*.
+    To assign a quality rating, follow the [QA/QC criteria]() stated below.
+- [ ] Download the rating file as a JSON and add it to the derivatives dataset.
+
+!!! warning "Immediately report images deemed *exclude*, as an issue in the dataset's repository"
+
+### Assessing functional images
+
+!!! danger "Insufficient quality of an fMRI run requires recalling the session"
+
+    - [ ] Immediately report images deemed *exclude*, as an issue in the dataset's repository.
+    - [ ] Proceed to scheduling an extra session after the initially-planned scanning period.
+
+#### RSfMRI
+- [ ] Open each *MRIQC* report on a current Web Browser (*Google Chrome* is preferred).
+- [ ] Visualize the first mosaic (BOLD average) and apply the [exclusion criteria](qaqc-criteria.md#bold-average)
+- [ ] Scroll down to the standard-deviation (std) mosaic and apply the [exclusion criteria](qaqc-criteria.md#standard-deviation-mosaic)
+- [ ] Scroll down to the background noise mosaic and apply the [exclusion criteria](qaqc-criteria.md#background-noise-mosaic)
+- [ ] Scroll down to the fMRI summary plot and apply the [exclusion criteria](qaqc-criteria.md#fmri-summary-plot) as well as [the exclusion criteria specific to the BHT](qaqc-criteria.md#resting-state).
+
+#### QCT
+- [ ] Open each *MRIQC* report on a current Web Browser (*Google Chrome* is preferred).
+- [ ] Visualize the first mosaic (BOLD average) and apply the [exclusion criteria](qaqc-criteria.md#bold-average)
+- [ ] Scroll down to the standard-deviation (std) mosaic and apply the [exclusion criteria](qaqc-criteria.md#standard-deviation-mosaic)
+- [ ] Scroll down to the background noise mosaic and apply the [exclusion criteria](qaqc-criteria.md#background-noise-mosaic)
+- [ ] Scroll down to the fMRI summary plot and apply the [exclusion criteria](qaqc-criteria.md#fmri-summary-plot) as well as [the exclusion criteria specific to the BHT](qaqc-criteria.md#quality-control-task) 
```

Comment:
I think we should look at the QCT the first:
- to train our eye
- to identify sessions where fMRI may have problems that went beyond the qct